versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -45,6 +45,8 @@ In certain situations, especially when the farmer terminates unexpectedly or enc
 
 Ensure to replace /path/to/your/plot with your actual plot path. Use this command cautiously as it modifies the plot state to recover from errors.
 
+<<<<<<< HEAD
```

Comment:
Are you aware of this merge conflict?

Reply:
I wasn't 🤦‍♂️ it's probably a good idea to check the changes through GitHub before requesting a re-review